lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,21 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    if (null !== $do = $this->getDigitalObject())
+    {
+      return $do->getChecksum();
+    }
+
+    return null;
```

Comment:
I'm not sure, I genuinely can't remember it ever coming up in CR! haha. I
don't mind either way.

Mike Gale,
Software Engineer, Artefactual Systems Inc.
http://www.artefactual.com | +1.604.527.2056

On Tue, Jun 21, 2016 at 3:01 PM, David Juhasz <notifications@github.com>
wrote:

> In lib/model/QubitInformationObject.php
> <https://github.com/artefactual/atom/pull/387#discussion_r67960472>:
>
> > @@ -1168,6 +1168,21 @@ public function getDigitalObject()
> >    }
> >
> >    /**
> > +   * Get the digital object's checksum value
> > +   *
> > +   * @return string  digital object checksum or null
> > +   */
> > +  public function getDigitalObjectChecksum()
> > +  {
> > +    if (null !== $do = $this->getDigitalObject())
> > +    {
> > +      return $do->getChecksum();
> > +    }
> > +
> > +    return null;
>
> Well, we didn't in my days as a developer. I don't know what you crazy
> kids get up to these days. 😜
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/artefactual/atom/pull/387/files/aa6b5386dce38ddaf87ee17df6b2d4b13a758875#r67960472>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0eEt1xoo6tmOqfy2ohO92qc_9AHwYLks5qOF9BgaJpZM4I20Pb>
> .
>